docs/hsi.xml

```diff
@@ -128,6 +128,13 @@
         connected, additional software to be installed, or disabling any existing
         security layers to measure.
       </para>
+      <para>
+        The HSI specification is primarily designed for consumer laptop, desktop
+        and server hardware, although some tests <emphasis>may</emphasis> still
```

Comment:
I don't necessarily agree on the server hardware aspect.  I haven't seen a single thing that talks about a BMC, or looking for surprise removal events or anything else server-y.

Reply:
Something like

        The HSI specification is primarily designed for consumer laptop and desktop
        hardware, although some tests <emphasis>may</emphasis> still make sense on
        server or embedded hardware.
        It is not expected that non-consumer hardware will publish an HSI number.

Reply:
that's better to me

Reply:
Fixed.